invoke.go

```diff
@@ -154,11 +156,20 @@ func sendEvent(ctx context.Context, route string, m InvokeMessage, t http.RoundT
 		resp = evt.String()
 	}
 
+	// return metdata about the CloudEvent that just so happens to be the same
+	// data structure as HTTP headers such that they are interchangeable at the
+	// higher level of the Invoke... invocation.
+	meta = make(map[string][]string)
+	meta["id"] = []string{evt.ID()}
+	meta["type"] = []string{evt.Type()}
+	meta["source"] = []string{evt.Source()}
+	meta["subject"] = []string{evt.Subject()}
+
```

Comment:
I think here we should probably just return an empty map, or one that includes all headers not prefixed with `ce-` (the prefix used for the event attributes in the header). The reason I had not already contributed this fix is because it was not immediately obvious to me how to get the headers when receiving an event with the golang library.

The string representation of a CloudEvent is already nicely formatted and displays all the event attributes. For example:

```
❯ func invoke -v
Context Attributes,
  specversion: 1.0
  type: echo
  source: function.eventViewer
  id: b9a722da-9dd7-4754-9eb7-f0cb1651433b
  time: 2022-04-05T19:47:18.518Z
  datacontenttype: application/json; charset=utf-8
Data,
  Hello World
```

Whereas with this addition, it ends up looking like this, which is difficult to read, incomplete in the first section, and duplicative.

```
~/demo/viewer is 📦 v0.1.0 via  v16.14.0
❯ func invoke -v
"source":function.eventViewer
"subject":
"id":40ee7efd-a6eb-44e6-9df7-f3d1fb56a430
"type":echo

Context Attributes,
  specversion: 1.0
  type: echo
  source: function.eventViewer
  id: 40ee7efd-a6eb-44e6-9df7-f3d1fb56a430
  time: 2022-04-05T20:00:50.653Z
  datacontenttype: application/json; charset=utf-8
Data,
  Hello World
```